MekHQ/src/mekhq/campaign/stratcon/StratconContractDefinition.java

```diff
@@ -0,0 +1,418 @@
+/*
+ * Copyright (c) 2019 The Megamek Team. All rights reserved.
+ *
+ * This file is part of MekHQ.
+ *
+ * MekHQ is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MekHQ is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MekHQ.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package mekhq.campaign.stratcon;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.namespace.QName;
+import javax.xml.transform.Source;
+
+import mekhq.MekHQ;
+import mekhq.MekHqConstants;
+import mekhq.MekHqXmlUtil;
+
+/**
+ * This class holds data relevant to the various types of contract
+ * that can occur in the StratCon campaign system.
+ * @author NickAragua
+ *
+ */
+@XmlRootElement(name="StratconContractDefinition")
+public class StratconContractDefinition {
+    public static final String ROOT_XML_ELEMENT_NAME = "ScenarioTemplate";
+    public static final int COUNT_SCALED = -1;
+    
+    private static ContractDefinitionManifest definitionManifest;
+    private static Map<Integer, StratconContractDefinition> loadedDefinitions = new HashMap<>();
+    
+    private static ContractDefinitionManifest getContractDefinitionManifest() {
```

Comment:
Odd to have a method in the middle of the variable declarations

Reply:
statics appear to be grouped together in this file.